src/main/java/org/jabref/logic/l10n/Localization.java

```diff
@@ -48,53 +95,108 @@ public static void setLanguage(String language) {
         }
     }
 
+    /**
+     * Public access to the messages bundle for classes like AbstractView.
+     * @return The internally cashed bundle.
+     */
+    public static LocalizationBundle getMessages() {
+        // avoid situations where this function is called before any language was set
+        if (locale == null) {
+            setLanguage("en");
+        }
+        return localizedMessages;
+    }
+
+    /**
+     * Creates and caches the language bundles used in JabRef for a particular language. This function first loads
+     * correct version of the "escaped" bundles that are given in {@link l10n}. After that, it stores the unescaped
+     * version in a cached {@link LocalizationBundle} for fast access.
+     * @param locale Localization to use.
+     */
     private static void createResourceBundles(Locale locale) {
-        messages = ResourceBundle.getBundle(RESOURCE_PREFIX, locale, new EncodingControl(StandardCharsets.UTF_8));
-        menuTitles = ResourceBundle.getBundle(MENU_RESOURCE_PREFIX, locale, new EncodingControl(StandardCharsets.UTF_8));
+        ResourceBundle messages = ResourceBundle.getBundle(RESOURCE_PREFIX, locale, new EncodingControl(StandardCharsets.UTF_8));
+        ResourceBundle menuTitles = ResourceBundle.getBundle(MENU_RESOURCE_PREFIX, locale, new EncodingControl(StandardCharsets.UTF_8));
+        // Just for the case something really stupid happened.
+        if (messages == null || menuTitles == null) {
+            LOGGER.error("Could not load language translation files in " + Localization.class);
```

Comment:
Same remark regarding the class files as by @Siedlerchr above

Reply:
Done.